tanjun/parsing.py

```diff
@@ -85,9 +85,7 @@
 class ParseableProto(typing.Protocol):
     """Protocol of a command which supports this parser interface."""
 
-    # This fucks with MyPy even though at runtime python just straight out ignores slots when considering protocol
-    if not typing.TYPE_CHECKING:  # compatibility.
-        __slots__ = ()
```

Comment:
this still causes issues with mypy's latest release so this change should be reverted

Reply:
Yep, didn't notice as I was not checking against the examples...